include/yaml-cpp/parser.h

```diff
@@ -47,7 +47,7 @@ class YAML_CPP_API Parser : private noncopyable {
   void Load(std::istream& in);
 
   /**
-   * Handles the next document by calling events on the {@param eventHandler}.
+   * Handles the next document by calling events on the { @param eventHandler }.
```

Comment:
Since this seems to be an incorrect way to refer to a parameter (I can't seem to find any indication it's even allowed to refer to a @param inline like this), can you just change it to @code? I don't want to special-case this one for having a space at the end.